### DISCUSSION TITLE
code-server can not run “turtle”

### DISCUSSION THREAD

**Discussion by wabbyy** [coder/code-server#3063] [Q&A]
error：

Traceback (most recent call last):
  File "/root/AJ/python/1.py", line 2, in <module>
    import turtle
  File "/usr/local/lib/python3.9/turtle.py", line 107, in <module>
    import tkinter as TK
  File "/usr/local/lib/python3.9/tkinter/__init__.py", line 37, in <module>
    import _tkinter # If this fails your Python may not be configured for Tk
ModuleNotFoundError: No module named '_tkinter'

**Comment by oxy** [comment]
1. Are you running `code-server` remotely on a server? We don't forward windows over X or anything of the sort, and only forward the editor, so tkinter wouldn't work.
2. If you're running locally, do you have `tcl`, `tk` and the other dependencies for `tkinter` installed?

**Reply by wabbyy** [reply]
i am running remotely on a server. 
so, will code-server update  to make it work in future? or is there solution of this?
thanks a lot!

**Reply by code-asher** [reply]
I don't think it'll be something we add to code-server, at least not any time soon.

You can forward X yourself with something like VNC though. 

Related: https://github.com/cdr/code-server/discussions/2090#discussioncomment-79375
